stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/MessageListView.kt

```diff
@@ -801,6 +838,18 @@ public class MessageListView : ConstraintLayout {
         public fun onThreadEntered(message: Message)
     }
 
+    public fun setOnAttachmentReplyOptionClickHandler(handler: AttachmentGalleryActivity.AttachmentReplyOptionHandler) {
+        this.onAttachmentReplyOptionClickHandler = handler
+    }
+
+    public fun setOnAttachmentShowInChatOptionClickHandler(handler: AttachmentGalleryActivity.AttachmentShowInChatOptionHandler) {
+        this.onAttachmentShowInChatOptionClickHandler = handler
+    }
+
+    public fun setOnAttachmentDeleteOptionClickHandler(handler: AttachmentGalleryActivity.AttachmentDeleteOptionHandler) {
+        this.onAttachmentDeleteOptionClickHandler = handler
+    }
+
```

Comment:
These are inserted here in the middle of the Listener interface declarations, they should be end the end of the handler setters

Reply:
Sorry, I've missed some rebase glitch here.